### ISSUE TITLE
Session hijacking attack

### ISSUE THREAD

**Issue by pbonito** [istio-ecosystem/authservice#262]
Hi team,
we found a problem related to session cookie generated by authservice (__Host-AIS_session).
- Login with 2 different account (UserA,UserB) from 2 different browser
- Replace the cookie value of UserB with one from UserA
- Refresh browser of UserB
- Active session is transferred from UserA to UserB

How we can avoid this? Is there a way to tie __Host-AIS_session to browser cookie?

Thanks in advance for your help

**Comment by nacx** [comment]
I'm not fully understanding the problem:
>Replace the cookie value of UserB with one from UserA

**Comment by pbonito** [comment]
Yes, it was a manual thing, but we agreed with security team that there was  not a valid solution to this issue.
